beacon-chain/core/helpers/attestation_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	types "github.com/prysmaticlabs/eth2-types"
 	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
-	"github.com/prysmaticlabs/prysm/beacon-chain/state/v1"
+	v1 "github.com/prysmaticlabs/prysm/beacon-chain/state/v1"
```

Comment:
Go check complained about incorrect order of imports and goimports automatically added the alias. Not sure why it was not necessary before.